game-core/src/main/java/games/strategy/engine/chat/Chat.java

```diff
@@ -43,7 +43,7 @@
   // initialization
   // synchronizes access to queue
   private final Object mutexQueue = new Object();
-  private List<Runnable> queuedInitMessages = new ArrayList<>();
+  private List<Runnable> queuedInitMessages;
```

Comment:
It looks like `queuedInitMessages` is used in the constructor:

```java
    queuedInitMessages.forEach(Runnable::run);
    assignNodeTags(chatters);
    queuedInitMessages = null;
```

The usage is useless, but probably should be removed to avoid an NPE.  (Connecting to the lobby will demonstrate this.)

Actually, you may want to review this again.  The other usages of this field in this class appear strange.  There may be some concurrency going on here that makes it appear this field is not actually being used.